src/jsifier.js

```diff
@@ -271,10 +271,13 @@ function ${name}(${args}) {
           // (not useful to warn/error multiple times)
           LibraryManager.library[symbol + '__docs'] = '/** @type {function(...*):?} */';
         } else {
-          const target = `Module['${mangled}']`;
+          let target = `wasmImports['${symbol}']`;
+          if (ASYNCIFY) {
+            target = `asyncifyStubs['${symbol}']`;
```

Comment:
Other places in the code seem to suggest that a symbol might or might not be in `asyncifyStubs`, but here we read from it unconditionally and overwrite the old value. Is that intentional?

Reply:
Here we generating the stub for missing `symbol`.   And we using `asyncifyStubs['${symbol}']` only within that stub function itself.

If this stub function is called, and `asyncifyStubs['${symbol}']` is still undefined at that point in time, that would be critical error and the program will abort.   In debug builds we show the assert below in that case.

Reply:
I see, sounds good. In that case I wrote a suggestion to revise the code slightly.